spec/recipes/server_install_spec.rb

```diff
@@ -3,8 +3,7 @@
 describe 'rundeck::server_install' do
   # ordered list of included recipes
   let(:included_recipes) do
-    %w(rundeck::default rundeck::_data_bags rundeck::_connect_rundeck_api_client
-       rundeck::_projects)
+    %w(rundeck::default rundeck::_data_bags)
   end
```

Comment:
you need to add specs for the other changes you made to the `server_install` recipe